inst/qml/jaspPlotBuilder.qml

```diff
@@ -0,0 +1,2932 @@
+                            { label: "Discrete: Apple", value: "colors_discrete_apple" },
+                            { label: "Discrete: Friendly Long", value: "colors_discrete_friendly_long" },
+                            { label: "Discrete: Okabe Ito", value: "colors_discrete_okabeito" },
+                            { label: "Discrete: IBM", value: "colors_discrete_ibm" },
+                            { label: "Discrete: Metro", value: "colors_discrete_metro" },
+                            { label: "Discrete: Candy", value: "colors_discrete_candy" },
+                            { label: "Discrete: Alger", value: "colors_discrete_alger" },
+                            { label: "Discrete: Rainbow", value: "colors_discrete_rainbow" },
+                            { label: "Continuous: Viridis", value: "colors_continuous_viridis" },
+                            { label: "Continuous: Magma", value: "colors_continuous_magma" },
+                            { label: "Continuous: Inferno", value: "colors_continuous_inferno" },
+                            { label: "Continuous: Plasma", value: "colors_continuous_plasma" },
+                            { label: "Continuous: Cividis", value: "colors_continuous_cividis" },
+                            { label: "Continuous: Rocket", value: "colors_continuous_rocket" },
+                            { label: "Continuous: Mako", value: "colors_continuous_mako" },
+                            { label: "Continuous: Turbo", value: "colors_continuous_turbo" },
+                            { label: "Continuous: Blue Pink Yellow", value: "colors_continuous_bluepinkyellow" },
+                            { label: "Diverging: Blue to Red", value: "colors_diverging_blue2red" },
+                            { label: "Diverging: Blue to Brown", value: "colors_diverging_blue2brown" },
+                            { label: "Diverging: BuRd", value: "colors_diverging_BuRd" },
+                            { label: "Diverging: BuYlRd", value: "colors_diverging_BuYlRd" },
+                            { label: "Diverging: Spectral", value: "colors_diverging_spectral" },
+                            { label: "Diverging: Icefire", value: "colors_diverging_icefire" }
+                        ]
+                    }
+                }
+
+                Group {
+                    columns:2
+                    Group {
+                        columns:1
+                        TextField {
+                            name: "customColors"
+                            label: qsTr("Custom colors")
+                            placeholderText: qsTr("e.g. red, blue, ##ff5733")
+                            fieldWidth: 150
+                        }
+
+
+
+                    Label {
+                        text: qsTr("Note: For available colors, see <a href='https://r-charts.com/colors/' style='color: blue; text-decoration: underline;'>this page</a>.")
```

Comment:
```suggestion
                        text: qsTr("Note: For available colors, see %1 this page %2").arg("<a href='https://r-charts.com/colors/' style='color: blue; text-decoration: underline;'>").arg("</a>.")
```

Reply:
We can do this to prevent translators from accidentally destroying its integrity.